src/site/content/en/blog/notification-triggers/index.md

```diff
@@ -0,0 +1,261 @@
+
+Request a [token](https://developers.chrome.com/origintrials/) for your origin. Add the token to
+your pages, there are two ways to provide this token on any pages in your origin: Add an
+`origin-trial` `<meta>` tag to the `head` of any page. For example, this may look something like:
+`<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">` If you can configure your server, you
+can also provide the token on pages using an `Origin-Trial` HTTP header. The resulting response
+header should look something like: `Origin-Trial: TOKEN_GOES_HERE`
+
+### Feature detection
+
+You can find out if the browser supports Notification Triggers by checking for the existence of the
+`showTrigger` property.
+
+```js
+if ("showTrigger" in Notification.prototype) {
+  /* Notification Triggers supported */
+}
+```
+
+### Scheduling a notification
+
+Scheduling a notification is similar to showing a regular push notification, with the difference
+being that you need to pass a `showTrigger` condition property with a `TimestampTrigger` object as
+the value to the notification's options object.
+
+```js
+const createScheduledNotification = async (tag, title, timestamp) => {
+  const registration = await navigator.serviceWorker.getRegistration();
+  registration.showNotification(title, {
+    tag: tag,
+    body: "This notification was scheduled 30 seconds ago",
+    showTrigger: new TimestampTrigger(timestamp + 30 * 1000)
+  });
+};
+```
+
+{% Aside %}
+  On desktop, Notifications Triggers will fire only if Chrome is running. On Android,
+  Notifications Triggers will fire completely independent from Chrome.
+{% endAside %}
+
+### Canceling a previously scheduled notification
+
+To cancel previously scheduled notifications, first request a list of all notifications that match a
+certain tag through `ServiceWorkerRegistration.getNotifications()`. Note that you need to pass the
+`includeTriggered` flag for scheduled notifications to be included in the list.
+
+```js
+const tag = 'meeting';
+const cancelScheduledNotification = async (tag) => {
+  const registration = await navigator.serviceWorker.getRegistration();
+  const notifications = await registration.getNotifications({
+    tag: tag,
+    includeTriggered: true,
+  });
+
+  if (Array.isArray(notifications)) {
+    notifications.forEach((notification) => notification.close());
+  }
+};
+```
+
+### Debugging
+
+You can use Chrome DevTools' Notifications debug service on the Application tab. To start debugging,
+press the small "Start recording events" button in the top-left corner (the circle), or hit
+'Command/Control' + 'E'. This allows you to both examine scheduled notifications, as well as see
+displayed and closed notifications. The screenshots below show this in action.
+
+<figure class="w-figure w-figure--fullbleed">
+  <img src="devtools-scheduled.png" alt="">
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Fig. 1 — Chrome DevTools showing the "scheduled" state of a notification.
```

Comment:
@kaycebasques I looked at the handbook. I don't think we've standardized on whether and how to number figures. By the way, I don't care how we do this. Just tell me what to do.

Reply:
Pretty sure that we're not numbering figures anymore. I created #1839 to fix the demo article. I will push an update to fix this formatting myself.